main.js

```diff
@@ -97,9 +120,13 @@ const Expander = function (el, opts) {
 
 	this.contentEl = this.el.querySelector('.' + this.contentClassName);
 
-	this.toggles = [].slice.apply(this.el.querySelectorAll(this.opts.toggleSelector));
+	if(!this.accordion) {
+		this.toggles = [].slice.apply(this.el.querySelectorAll(this.opts.toggleSelector));
```

Comment:
I think we can replace `[].slice.apply` with `Array.from` as it is in most browsers and is also in the default set of polyfills served from polyfill.io